test/integration/read-write-concern/write_concern.test.js

```diff
@@ -1,10 +1,10 @@
-'use strict';
-
 const { expect } = require('chai');
-const { LEGACY_HELLO_COMMAND } = require('../../../src/constants');
 
-const mock = require('../../tools/mongodb-mock/index');
 const { MongoClient } = require('../../../src');
+const { LEGACY_HELLO_COMMAND } = require('../../../src/constants');
```

Comment:
Should I go ahead and convert this file to a typescript file?

Reply:
Sure!